File: pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.github.pagehelper.PageInfo;
import com.pinyougou.pojo.TbBrand;
import com.pinyougou.pojo.TbBrandExample;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {

    PageResult findAll(TbBrandExample example, int page, int size);

    List<TbBrand> findAll(TbBrandExample example);

    TbBrand findById(Long id);

    void save(TbBrand tbBrand);

    void update(TbBrand tbBrand);

    void delete(Long id);

    PageResult findPage(int pageNumber, int pageSize);

    List<Map> selectBrandOptions();

}

File: pinyougou_manager_web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {

    this.findPage=function (pageNumber,pageSize,searchEntity) {
        return $http.post("../brand/findPageCondition.do?pageNumber="+pageNumber+
            "&pageSize="+pageSize,searchEntity);
    };

    this.add=function (entity) {
        return $http.post("../brand/add.do",entity);
    };

    this.update=function (entity) {
        return $http.post("../brand/update.do",entity);
    };

    this.findOne=function (id) {
        return $http.get("../brand/findOne.do?id="+id);
    };

    this.dele=function (ids) {
        return $http.get("../brand/delete.do?ids="+ids);
    };
    this.selectBrandOptions=function (){
        return $http.get("../brand/selectBrandOptions.do");
    }
});